client/my-sites/comments/controller.js

```diff
@@ -12,27 +12,57 @@ import route from 'lib/route';
 import CommentsManagement from './main';
 import controller from 'my-sites/controller';
 
-export const comments = function( context ) {
-	const siteSlug = route.getSiteFragment( context.path );
-	const status = context.params.status === 'pending' ? 'unapproved' : context.params.status;
+export const isStatusValid = status => {
+	const validStatuses = [ 'pending', 'approved', 'spam', 'trash', 'all' ];
+	return validStatuses.indexOf( status ) !== -1;
```

Comment:
we do tend to use more semantic expression for set membership…

```js
import { includes } from 'lodash';

return includes( validStatuses, status );
```

this is simply to communicate more expressively in the code what our intention is. `indexOf()` communicates that we are looking for an index, which we aren't.

feel free to make your own decision on this.